docs/CONTRIBUTING.md

```diff
@@ -3,47 +3,63 @@ Thank you for contributing to Little Light! All suggestions and feature requests
 Here are some tips to make sure your pull request can be merged smoothly:
 
 1. If you want to add a feature or make some change to Little Light, consider [filing an issue](https://github.com/DestinyItemManager/DIM/issues/new) describing your idea first. This will give the Little Light community a chance to discuss the idea, offer suggestions and pointers, and make sure what you're thinking of fits with the style and direction of Little Light. If you want a more free-form chat, [join our Discord](https://discord.gg/dTKhBD).
-1. Resist the temptation to change more than one thing in your PR. Keeping PRs focused on a single change makes them much easier to review and accept. If you want to change multiple things, or clean up/refactor the code, make a new branch and submit those changes as a separate PR.
-1. Little Light is written in [Dart](https://www.dartlang.org/) and built in [Flutter](https://www.flutter.io)
+2. Resist the temptation to change more than one thing in your PR. Keeping PRs focused on a single change makes them much easier to review and accept. If you want to change multiple things, or clean up/refactor the code, make a new branch and submit those changes as a separate PR.
+3. Little Light is written in [Dart](https://www.dartlang.org/) and built in [Flutter](https://www.flutter.io)
 
 ## Developer Quick start
-
+<!-- no toc  -->
```

Comment:
This is so the [Markdown All-in-One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) extension doesn't keep breaking the table of contents.